Assets/multilingual font compression (#1322)
massgov/mayflower: packages/core/stories/tokens/typography/Typography.stories.mdx

  }
</div>

> ## Language support

Reviewer: ```suggestion
## Language Support
```

The following languages are officially supported by [Mass.gov](https://www.mass.gov):

Arabic, Armenian, Chinese (Simplified), Filipino, French, German, Greek, Gujarati, Haitian Creole, Hebrew, Hindi, Italian, Japanese, Khmer, Korean, Lao, Malayalam, Persian, Polish, Portuguese, Russian, Spanish, Tamil, Telugu, Thai, Ukrainian, Urdu, and Vietnamese.

The foolowing do not yet have webfont support: Chinese (Simplified), Japanese, and Korean.

If your project needs to support an addtional language, please [submit a feature request](https://github.com/massgov/mayflower/issues/new/choose) on Github.

> ### Languages by script

Reviewer: ```suggestion
### Languages by Script
```

---

> #### Left-to-right

Reviewer: ```suggestion
### Left-to-right
```

#### Armenian script

<p className="lang-label">Armenian: </p>

> <p lang="hy" className="font-specimen">Ու նայիրեան աղջիկների հեզաճկուն պա՛րն եմ սիրում։</p>

Reviewer: what is this text? can we revert it to the original text -- which is the title of the mass.gov homepage?

<!-- @todo add support
#### Chinese, simplified script

<p className="lang-label">Chinese: </p>

> <p lang="zh-Hans" className="font-specimen">微風迎客，軟語伴茶</p>

Reviewer: see comment above, same for all the other languages

  }
</div>

## Language Support

The following languages are officially supported by [Mass.gov](https://www.mass.gov):

Arabic, Armenian, Chinese (Simplified), Filipino, French, German, Greek, Gujarati, Haitian Creole, Hebrew, Hindi, Italian, Japanese, Khmer, Korean, Lao, Malayalam, Persian, Polish, Portuguese, Russian, Spanish, Tamil, Telugu, Thai, Ukrainian, Urdu, and Vietnamese.

> The foolowing do not yet have webfont support: Chinese (Simplified), Japanese, and Korean.

Reviewer: ```suggestion
The following do not yet have Noto Sans webfont support: Chinese (Simplified), Japanese, and Korean.
```

#### Armenian script

<p className="lang-label">Armenian: </p>

> <p lang="hy" className="font-specimen">Բարի գալուստ Մասաչուսեթս</p>

Reviewer: My original comment was about reverting all the font specimen text changes in this file, not just for these two languages. Can you revert all of them instead?

Reviewer: The specimen text should match what's in production --> https://mayflower.digital.mass.gov/core/index.html?path=/docs/foundation-typography--page

<p lang="pl" className="font-specimen">Witamy w Massachusetts</p>

<p className="lang-label">Portuguese: </p>

> <p lang="es" className="font-specimen">Bienvenido a massachusetts</p>

Reviewer: ```suggestion
<p lang="pt" className="font-specimen">Bem vindo a Massachusetts</p>
```

Reviewer: fix portuguese

#### Thai script

<p className="lang-label">Thai: </p>
<p lang="Thai" className="font-specimen">เป็นมนุษย์สุดประเสริฐเลิศคุณค่า กว่าบรรดาฝูงสัตว์เดรัจฉาน จงฝ่าฟันพัฒนาวิชาการ อย่าล้างผลาญฤๅเข่นฆ่าบีฑาใคร ไม่ถือโทษโกรธแช่งซัดฮึดฮัดด่า หัดอภัยเหมือนกีฬาอัชฌาสัย ปฏิบัติประพฤติกฎกำหนดใจ พูดจาให้จ๊ะ ๆ จ๋า ๆ น่าฟังเอยฯ</p>

> #### Right-to-Left

Reviewer: ```suggestion
### Right-to-Left
```

<p lang="fa" className="font-specimen">به ماساچوست خوش آمدید</p>

<p className="lang-label rtl">Urdu: </p>

> <p lang="fa" className="font-specimen">به ماساچوست خوش آمدید</p>

Reviewer: ```suggestion
<p lang="ur" className="font-specimen">میساچوسٹس میں آپ کا استقبال ہےد</p>
```
